### PR TITLE
Map attribution option

### LINKED ISSUE #67: Geocoding map attribution should be an option
Since using the ArcGIS Online Geocoding service is an option the [map attribution text](https://github.com/Esri/esri-leaflet-geocoder/blob/8ace0a6e605a3b0b8047f28a8e1340638c79ff70/src/Controls/Geosearch.js#L237-L239) should also be an option in the constructor. The default option could remain "Geocoding by Esri".

### PR DESCRIPTION
**Pull request by npeihl**
Fixes #67

### CODE REVIEW

File: src/Providers/FeatureLayer.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ EsriLeafletGeocoding.Controls.Geosearch.Providers.FeatureLayer = L.esri.Services
     }
   },
   initialize: function(url, options){
-    L.esri.Services.FeatureLayer.prototype.call(this, url, options);
+    L.esri.Services.FeatureLayer.prototype.initialize.call(this, url, options);
```
Comment (jgravois) [inline src/Providers/FeatureLayer.js]: forgive me if i'm being dense, but what does this change in the FeatureLayer provider accomplish?

**Comment by jgravois** [comment]
thanks for this @npeihl.  i hate to be nitpicky, but we'll need to include a note in the documentation that explains that altering/omitting the geocoding attribution is only allowed if you are **not** using the World Geocoding Service.

File: src/Providers/FeatureLayer.js (hunk 1 above)
Comment (npeihl) [inline src/Providers/FeatureLayer.js]: The tests and Travis build are failing on the master because the call function is undefined. It appears to me that it was a typo and 'call' should have been called on the initialize function. I successfully ran the tests and the Travis build is now successful, too.

**Comment by npeihl** [comment]
@jgravois Is it enough to put that into documentation? I suppose we could build a check into the function that ignores custom text in mapAttribution and forces the default if the 'useArcgisWorldGeocoder' option is true.

**Comment by jgravois** [comment]
thats not a bad idea.  the doc can explain that `mapAttribution` is optional and will override the default ifaof `useArcgisWorldGeocoder` is set to `false`.

File: src/Providers/FeatureLayer.js (hunk 1 above)
Comment (jgravois) [inline src/Providers/FeatureLayer.js]: i kinda suspected this was an unrelated improvement.  thanks for clarifying.

Comment (npeihl) [inline src/Providers/FeatureLayer.js]: Would it have been better to submit as a different PR?

Comment (jgravois) [inline src/Providers/FeatureLayer.js]: not at all.  just keep laying down commits here until we're all happy.

**Comment by npeihl** [comment]
@jgravois I've updated the PR and tested that it works. Let me know if you need anything else.

**Comment by jgravois** [comment]
thanks for this @npeihl!
